MAUI/Maps/migration.md

```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
+    layer.ShapesSource = MapSource.FromUri(new Uri("https://cdn.syncfusion.com/maps/map-data/world-map.json"));
+    MapPolygonLayer mapPolygonLayer = new MapPolygonLayer();
+    MapPolygon polygon1 = new MapPolygon();
+    polygon1.Points = new ObservableCollection<MapLatLng>()
+    {
+    new MapLatLng(37.6173, 55.7558),
+    new MapLatLng(87.1216, 53.7596),
+    new MapLatLng(105.3188, 61.5240)
+    };
+    polygon1.Fill = Colors.Blue;
+    polygon1.Stroke = Colors.DarkBlue;
+    polygon1.StrokeThickness = 4;
+    mapPolygonLayer.Polygons.Add(polygon1);
+    layer.Sublayers.Add(mapPolygonLayer);
+    maps.Layer = layer;
+    this.Content = maps;
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+### Polyline Layer
+
+<table>
+<tr>
+   <th>Xamarin SfMaps</th>
+   <th>.NET MAUI SfMaps</th>
+</tr>
+<tr>
+<tr>
+   <td>
+      {{'[ShapeType](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeFileLayer.html#Syncfusion_SfMaps_XForms_ShapeFileLayer_ShapeTypeProperty)'| markdownify }}
+   </td>
+   <td>
+      {{'MapPolyline'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[Points](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeFileLayer.html#Syncfusion_SfMaps_XForms_ShapeFileLayer_PointsProperty)'| markdownify }}
+   </td>
+   <td>
+      {{'Points'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[ShapeFill](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeFill)'| markdownify }} in
+       {{'[ShapeSetting](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html)'| markdownify }} class
+   </td>
+   <td>
+      {{'[-](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapShapeLayer.html#Syncfusion_Maui_Maps_MapShapeLayer_ShapeFill)'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[ShapeStroke](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeStroke)'| markdownify }} in
+       {{'[ShapeSetting](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html)'| markdownify }} class
+   </td>
+   <td>
+      {{'Stroke'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[ShapeStrokeThickness](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeStrokeThickness)'| markdownify }} in
+       {{'[ShapeSetting](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html)'| markdownify }} class
+   </td>
+   <td>
+      {{'StrokeThickness'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[-]'| markdownify }}
+   </td>
+   <td>
+      {{'StrokeDashArray'| markdownify }}
+   </td>
+</tr>
+</table>
+
+The following code example explains how to intialize the layer and customize the layer in Xamarin SfMaps and .Net MAUI SfMaps.
```

Comment:
The following code example explains how to initialize and customize the layer in the Xamarin SfMaps and .Net MAUI SfMaps.

Reply:
Modified